faqs.rst

```diff
@@ -1,35 +1,60 @@
 .. _faqs:
 
-==========
+####
 FAQs
-==========
+####
+
 Some frequently asked questions about flux and their answers.
 
 .. _flux_run_mac:
 
------------------------
+
+*****************
+General Questions
+*****************
+
+What's with the fancy ƒ?
+========================
+
+Flux job IDs and their multiple encodings are described in
+`RFC 19 <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_19.html>`_. The default F58 encoding assumes that your terminal session can
+handle UTF-8.
```

Comment:
I wonder if we should state something like: The UTF-8 `ƒ` character is used to denote a Flux jobid in F58 encoding when the current locale supports UTF-8 multi-byte characters. When the locale does not support UTF-8, then an ASCII `f` is used. The ASCII rendering of jobids can be forced by setting ``FLUX_F58_FORCE_ASCII=1`` in your environment.

Actually, the main point is that Flux will try to first determine if the current locale supports UTF-8 multi-byte characters before using `ƒ`. Maybe we just say that? Up to you.

Reply:
Great point.  I'll mention something to that effect.

Reply:
I forced an update - look OK?

Reply:
Yes, nice!